pkg/agent/proxy/proxier_test.go

```diff
@@ -162,7 +162,8 @@ func testClusterIP(t *testing.T, svcIP net.IP, epIP net.IP, isIPv6 bool, extraSv
 	defer ctrl.Finish()
 	mockOFClient := ofmock.NewMockClient(ctrl)
 	mockRouteClient := routemock.NewMockInterface(ctrl)
-	fp := NewFakeProxier(mockRouteClient, mockOFClient, nil, isIPv6, withProxyAll)
+	mockGroupAllocator := ofmock.NewMockGroupAllocator(ctrl)
```

Comment:
groupAllocator has no external dependency and is simple to predict its return value. Do we need to mock it and generate the mocking implementation? I feel it's simpler to just pass a real instance to be close to real runtime and avoid code of mocking the return value.

Reply:
Updated.